### ISSUE TITLE
installing opentelemetry-id-generator-aws-xray fails due to type error

### ISSUE THREAD

**Issue by dradetsky** [aws-observability/aws-otel-js#25]
As soon as I run

```
npm i @aws/otel-aws-xray-id-generator
```

I start seeing the following errors in my build

```
node_modules/@types/mocha/index.d.ts:2734:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'beforeEach' must be of type 'Lifecycle', but here has type 'HookFunction'.

2734 declare var beforeEach: Mocha.HookFunction;
                 ~~~~~~~~~~

  node_modules/@types/jest/index.d.ts:35:13
    35 declare var beforeEach: jest.Lifecycle;
                   ~~~~~~~~~~
    'beforeEach' was also declared here.
```

and several other similar errors.

My project does have `jest` & `@types/jest`. It does not have mocha. Looking at the package file, I see

```
  "dependencies": {
    ....
    "@types/mocha": "7.0.2",
    "@types/node": "12.12.47",
    "@types/sinon": "9.0.4",
    "@types/webpack-env": "1.15.2",
    ....
```

Were these supposed to be `devDependencies`?

**Comment by dradetsky** [comment]
@anuraaga I know you merged my PR, but it would be really helpful if this could be released.

**Comment by KKelvinLo** [comment]
@dradetsky Sorry for the trouble, we will be releasing this shortly.

**Comment by dradetsky** [comment]
@KKelvinLo is there some reason you can't just publish the tip of master immediately as `0.12.1`? I mean, the package is broken for anyone who uses both jest (9,514,103 weekly downloads) and typescript, and there's barely any other changes. Hell, even a prerelease version (`0.12.0-0`) would be better than nothing.

**Comment by KKelvinLo** [comment]
@dradetsky Unfortunately it is due to an issue with NPM ... we filed a couple tickets and followed up without a response.

**Comment by KKelvinLo** [comment]
Tip of master published as 0.12.1 - thanks for your patience
